packages/api/tests/serial/endpoints/test-granules.js

```diff
@@ -518,7 +519,9 @@ test.serial('DELETE deleting an existing unpublished granule', async (t) => {
   }
 
   // create a new unpublished granule
-  await granuleModel.create(newGranule);
+  const granuleRecord = await granuleModel.create(newGranule);
+  await indexer.indexGranule(esClient, granuleRecord, esIndex);
```

Comment:
Why does this test need to index to ES?

Reply:
Because it creates a new Granule in line 522 and they need to have it in ES because the delete looks for it there to delete it.